runtime/src/accounts_db.rs

```diff
@@ -10382,11 +10383,32 @@ pub mod tests {
                 vec![write_version],
             );
         let old_written = storage.written_bytes();
-        storage.accounts.append_accounts(&storable_accounts, 0);
+        let offsets = storage
+            .accounts
+            .append_accounts(&storable_accounts, 0)
+            .unwrap();
```

Comment:
just a note to myself as I was reviewing:

new unwrap here; this seems fine as we don't expect appending to fail for any reason here (we always have at least one account).

Reply:
also in test code where a failed unwrap is useful to know about.